nativelib/src/main/resources/scala-native/gc/immix/WeakRefStack.c

```diff
@@ -0,0 +1,49 @@
+#include "WeakRefStack.h"
+#include "datastructures/Stack.h"
+#include "metadata/ObjectMeta.h"
+#include "headers/ObjectHeader.h"
+#include "State.h"
+#include <stdbool.h>
+
+extern word_t *__modules;
+int visited = false;
+
+// A collection of marked WeakReferences.
+// Used to correctly set "NULL" values in place of cleaned objects
+// and to call other handler functions with WeakRefStack_CallHandlers.
+
+void WeakRefStack_Init(size_t size) { Stack_Init(&weakRefStack, size); }
```

Comment:
Do you mean adding the `inline` modifier or removing those functions entirely and relying on Stack_*()? I was hoping this way it would be more readable, but I can change it of course.

Reply:
Now I noticed I ended up doing the exact opposite in commix (as in I use GreyList_*() directly there), so I will change immix according to your suggestion.